ui/src/components/ProvisionD-Config.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <TabMenu :model="TabMenuItem" :activeIndex="activeIndex"/>
+    <router-view></router-view>
+</template>
+<script setup lang="ts">
+
+import TabMenu from 'primevue/tabmenu';
+import { computed, ref } from 'vue';
```

Comment:
computed import unused and can be removed

Reply:
Removed the unused import